test/test-tlc-green-taxi-trip.rb

```diff
@@ -0,0 +1,59 @@
+class TLCGreenTaxiTripTest < Test::Unit::TestCase
+  def record(*args)
+    DatasetsParquet::TLC::GreenTaxiTrip::Record.new(*args)
+  end
+
+  test("each") do
+    dataset = DatasetsParquet::TLC::GreenTaxiTrip.new(year: 2022, month: 1)
+    records = dataset.each.to_a
+
+    assert_equal([
+                   62495,
+                   record(2,
+                          Time.parse('2022-01-01 09:14:21 +0900'),
+                          Time.parse('2022-01-01 09:15:33 +0900'),
+                          "N",
```

Comment:
`false`?

Reply:
Ah, `"N"` is processed in `TLC::GreenTaxiTrip::Record`.

How about comparing `records.first.to_h` instead of `records.first` to check value conversion?

Reply:
fix: [3dfe3a0](https://github.com/red-data-tools/red-datasets-parquet/pull/4/commits/3dfe3a0e92a68c1b661c7d9f752d62630e4b5f5c) Thank you so much. I fixed the test case of yellow taxi trip too.